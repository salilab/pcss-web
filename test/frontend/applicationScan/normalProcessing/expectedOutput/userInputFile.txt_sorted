A8MTL0
BARKAN
P01889
P01891
P01892
P01911
P03989
P04222
P04229
P04439
P05534
P10314
P10316
P10319
P10321
P13746
P13760
P13761
P16188
P16189
P16190
P18462
P18463
P18464
P18465
P20039
P27348
P30153
P30154
P30443
P30447
P30450
P30453
P30455
P30456
P30457
P30459
P30460
P30461
P30462
P30464
P30466
P30475
P30479
P30480
P30481
P30483
P30484
P30485
P30486
P30487
P30488
P30490
P30491
P30492
P30493
P30495
P30498
P30499
P30501
P30504
P30505
P30508
P30510
P30512
P30685
P31946
P31947
P55957
P61981
P62258
P63104
P63151
P79483
Q00005
Q03624
Q04826
Q04917
Q07000
Q09160
Q13362
Q14738
Q15172
Q15173
Q16537
Q29718
Q29836
Q29865
Q29940
Q29960
Q29963
Q30134
Q30167
Q31610
Q31612
Q4AC99
Q66LE6
Q95365
Q95604
Q96QU6
Q9TNN7
Q9TQE0
Q9Y2T4
