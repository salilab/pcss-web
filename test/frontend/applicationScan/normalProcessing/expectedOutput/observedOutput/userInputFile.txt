P31946
P62258
Q04917
P61981
P31947
P27348
P63104
P30443
P01892
P04439
P13746
Q96QU6
Q4AC99
P30447
P05534
P18462
P30450
P30512
P16188
P16189
P10314
P16190
P30453
P30455
P30456
P30457
P01891
P10316
P30459
Q09160
P01889
P30460
P30461
P30462
P30464
P30466
P03989
P30685
P18463
Q95365
P30475
Q04826
P30479
A8MTL0
P30480
P30481
P30483
P30484
P30485
P30486
P30487
P30488
P18464
P30490
P30491
P30492
P30493
P30495
P18465
P10319
Q29940
Q29836
Q31612
P30498
Q31610
Q29718
P30499
P30501
P04222
P30504
Q9TNN7
Q29963
P10321
P30505
P30508
P30510
Q07000
Q29960
Q95604
Q29865
Q15172
Q15173
Q14738
Q16537
Q13362
P30153
P30154
P63151
Q00005
Q66LE6
Q9Y2T4
P04229
P13760
P13761
Q30134
Q9TQE0
Q30167
P20039
P01911
P79483
P55957
Q03624
BARKAN
